### PR TITLE
Line docs

### PR DESCRIPTION
**Pull request by seejamescode**
Please review the readme for clarity on how to use!

### CODE REVIEW

File: packages/charts-line/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# IBM Charts - Line
+## @ibm-design/charts-line
+An implementation of the IBM Design Language's [line chart](https://www.ibm.com/design/language/experience/data-visualization/chart-models/linechart/) using [C3.js](http://c3js.org/).
+
+![Alt text](/charts-line.gif?raw=true "Optional Title")
```
Comment (rodet) [inline packages/charts-line/README.md]: There should be a comprehensible alt text and title here

File: packages/charts-line/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Usage
+This add-on to C3.js uses the exact same [arguements](http://c3js.org/reference.html). Be sure to add the `ibm-charts-line.css` and `ibm-charts-line.js` files after the D3 and C3 files are referenced:
```
Comment (rodet) [inline packages/charts-line/README.md]: Typo on "arguements"

File: packages/charts-line/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                ],
+              },
+              id: 'myChart',
+            });
+        </script>
+    </body>
+</html>
+```
+
+To use the [C3.js API](http://c3js.org/reference.html#api-focus), prefix the function with `ibmChart` and then your chart's unique identifier:
```
Comment (rodet) [inline packages/charts-line/README.md]: I feel the prefix wording here is a bit weird... Maybe it's just me as non native. Consider a wording like "to use the C3.js API, call it through ibmChart and your chart's unique identifier".

File: packages/charts-line/README.md (hunk 1 above)
Comment (rodet) [inline packages/charts-line/README.md]: The link to the gif seems broken when I look at the current branch state: https://github.com/IBM-Design/charts/tree/line-docs/packages/charts-line . Not sure if the `/` at the beginning of the link is needed.

**Review by rodet — COMMENTED** [review]
Hi James, can you have a look at those comments and comment back and change what you find useful? Looks like there are no blockers, so I'll approve by then.

**Comment by seejamescode** [comment]
Ready for rereview!

**Review by rodet — APPROVED** [review]
Perfect!

**Pull request merged**